sql/postgresql/PostgreSQLLexer.g4

```diff
@@ -27,90 +28,168 @@ https://github.com/tunnelvisionlabs/antlr4-grammar-postgresql/blob/master/src/co
  *    promote the sale, use or other dealings in this Software without prior
  *    written authorization from Tunnel Vision Laboratories, LLC.
  */
-
-
 lexer grammar PostgreSQLLexer;
-
 /* Reference:
  * http://www.postgresql.org/docs/9.3/static/sql-syntax-lexical.html
  */
 
-options { language = CSharp; }
 
-@header {
+
+options { language = CSharp; }
+@ header
+{
 using System.Collections.Generic;
 using System.Diagnostics;
 }
-
-@members {
+@ members
+{
 /* This field stores the tags which are used to detect the end of a dollar-quoted string literal.
  */
 private readonly Stack<String> _tags = new Stack<String>();
 }
-
 //
+
 // SPECIAL CHARACTERS (4.1.4)
+
 //
 
 // Note that Asterisk is a valid operator, but does not have the type Operator due to its syntactic use in locations
-// that are not expressions.
-
-Dollar			: '$';
-OPEN_PAREN		: '(';
-CLOSE_PAREN		: ')';
-OPEN_BRACKET		: '[';
-CLOSE_BRACKET	: ']';
-COMMA			: ',';
-SEMI    		: ';';
-COLON			: ':';
-STAR     		: '*';
-EQUAL			: '=';
-DOT			: '.';
-//NamedArgument	: ':=';
-PLUS            : '+';
-MINUS           : '-';
-SLASH           : '/';
-CARET           : '^';
-LT              : '<';
-GT              : '>';
-LESS_LESS       : '<<';
-GREATER_GREATER : '>>';
-COLON_EQUALS    : ':=';
-LESS_EQUALS     : '<=';
-EQUALS_GREATER  : '=>';
-GREATER_EQUALS  : '>=';
-DOT_DOT         : '..';
-NOT_EQUALS                           : '<>';
-TYPECAST            : '::';
-PERCENT : '%';
-PARAM : '$'([0-9])+;
 
+// that are not expressions.
 
-//
-// OPERATORS (�4.1.3)
-//
-
-// this rule does not allow + or - at the end of a multi-character operator
+Dollar
+   : '$'
+   ;
+
+OPEN_PAREN
+   : '('
+   ;
+
+CLOSE_PAREN
+   : ')'
+   ;
+
+OPEN_BRACKET
+   : '['
+   ;
+
+CLOSE_BRACKET
+   : ']'
+   ;
+
+COMMA
+   : ','
+   ;
+
+SEMI
+   : ';'
+   ;
+
+COLON
+   : ':'
+   ;
+
+STAR
+   : '*'
+   ;
+
+EQUAL
+   : '='
+   ;
+
+DOT
+   : '.'
+   ;
+   //NamedArgument	: ':=';
+   
+PLUS
+   : '+'
+   ;
+
+MINUS
+   : '-'
+   ;
+
+SLASH
+   : '/'
+   ;
+
+CARET
+   : '^'
+   ;
+
+LT
+   : '<'
+   ;
+
+GT
+   : '>'
+   ;
+
+LESS_LESS
+   : '<<'
+   ;
+
+GREATER_GREATER
+   : '>>'
+   ;
+
+COLON_EQUALS
+   : ':='
+   ;
+
+LESS_EQUALS
+   : '<='
+   ;
+
+EQUALS_GREATER
+   : '=>'
+   ;
+
+GREATER_EQUALS
+   : '>='
+   ;
+
+DOT_DOT
+   : '..'
+   ;
+
+NOT_EQUALS
+   : '<>'
+   ;
+
+TYPECAST
+   : '::'
+   ;
+
+PERCENT
+   : '%'
+   ;
+
+PARAM
+   : '$' ([0-9])+
+   ;
+   //
+   
+   // OPERATORS (?4.1.3)
```

Comment:
I suggest removing excess indents like this one. Also `?` looks weird here (probably it's here due to incorrect encoding conversion):

```suggestion
// OPERATORS (4.1.3)
```